core/src/main/java/org/elasticsearch/common/rounding/TimeZoneRounding.java

```diff
@@ -64,7 +64,9 @@ public Builder(TimeValue interval) {
         }
 
         public Builder timeZone(DateTimeZone timeZone) {
-            this.timeZone = timeZone;
+            if (timeZone != null) {
```

Comment:
Will add throwing an exception here, this was just for added security since the callers should already always use UTC as default.